python/sglang/srt/managers/scheduler.py

```diff
@@ -385,6 +416,25 @@ def event_loop_overlap(self):
 
             self.last_batch = batch
 
+    def loop_for_send_tree_cache(self):
+        while True:
+            self.send_tree_cache_to_queue()
+            time.sleep(1)
+
+    def send_tree_cache_to_queue(self):
+        if self.pre_radix:
+            try:
+                node = deepcopy(self.tree_cache.root_node)
+                send_data = RadixCacheSend(
+                    gpu_id=self.gpu_id, root_node=node, time=time.time()
```

Comment:
the radix cache will even contain GPU tensors. Please only send a simplified version without any GPU tensors.

Reply:
I check the pytorch doc and find that may be we can use torch.multiprocessing. How do you think of it?
https://pytorch.org/docs/stable/notes/multiprocessing.html
![image](https://github.com/user-attachments/assets/8d960912-ce7a-4731-a865-f96e52f1d674)

Reply:
`torch.multiprocessing` is not helpful here because we do not need to transfer any `values` in the radix tree.